tools/xkbcli-completion.sh

```diff
@@ -0,0 +1,97 @@
+# bash completion support for xkbcli.
+
+# See completion API documentation: https://github.com/scop/bash-completion
+# NOTE: The script parses the commands help messages to provide the completions,
+#       thus any new subcommand or option will be supported, as long as it has its
+#       entry in the help messages. This should result in low maintenancei effort.
+
+___xkbcli_main()
+{
+    # Initialization: https://github.com/scop/bash-completion/blob/fdf4456186eb4548ef628e65fb1be73d8e4695e9/bash_completion.d/000_bash_completion_compat.bash#L205
+    local cur prev words cword
+    _init_completion -s || return
+
+    local i=1 cmd
```

Comment:
```suggestion
    local cur prev words cword cmd
    _init_completion -s || return

    local i=1
```
Mixing uninitialized and initialized declarations seems messy

Reply:
Fixed